packages/x-components/src/design-system/utilities/line-clamp.scss

```diff
@@ -0,0 +1,8 @@
+@for $i from 2 through 6 {
+  .x-line-clamp--#{$i} {
+    overflow: hidden;
+    display: -webkit-box;
+    -webkit-box-orient: vertical;
+    -webkit-line-clamp: #{$i};
```

Comment:
As this is an utility class, should you add `!important` to some of the properties? I'm specially worried with the `display: -webkit-box`, we tend to override the display property.